source/source-connector/configuration-properties/kafka-topic.txt

```diff
@@ -38,8 +45,8 @@ Settings
      - | **Type:** string
        |
        | **Description:**
-       | The prefix to prepend to database and collection names to
-         generate the name of the Kafka topic on which to publish the data.
+       | The string to attach to the beginning of the change event
```

Comment:
Suggestion:

I think this description could be confusing since "attach" is rarely used to describe modifying a programming "string", but is commonly used to describe modifying a cord/rope. I think "concatenate" and "add" are more commonly used, if the intent is to avoid "prepend".

I think it is also helpful to mention that the prefix, database name, and collection name are separated by the character specified in "topic.separator". Maybe something like the following would be better and more similar to "topic.separator":

"
Specifies the beginning of the name of the destination Kafka topic to which the connector publishes change stream events. The destination topic name is composed of the ``topic.prefix`` value, database name, and collection name, separated by the value specified in the ``topic.separator`` setting.
"

Something similar applies to the suffix setting.